hack/eks

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: terraform enviroment variables need to be set
+INST_PROFILE_NAME=suwang-eks-worker-nodes-1-NodeInstanceProfile-8LL239W4W3GN
+ROLE_NAME=k8s
+
+AWS_CLI="aws"
+TERRAFORM="terraform"
+KUBECTL="kubectl"
+AWS_IAM="aws-iam-authenticator"
+RUN_PATH="$HOME/tmp/terraform-eks"
+CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/eks
+CMD_ARRAY=($AWS_CLI $TERRAFORM $KUBECTL $AWS_IAM)
+
+_usage="Usage: $0 : create eks cluster
+                [destroy] : destroy eks cluster
+                [kubectl] : access eks
+                [load] : load image to eks cluster
+                [--help|h]"
+
+function print_help {
+    echo "Try '$0 --help' for more information."
+}
+
+function print_usage {
+    echo "$_usage"
+}
+
+function validate {
+  for cmd in ${CMD_ARRAY[@]}; do
+     which $cmd > /dev/null 2>&1
+     if [ $? -ne 0 ]; then
+       echo $cmd not found, please install first
+       exit 1
+     fi
+  done
+
+  # check for aws permissions
+  $AWS_CLI eks list-clusters > /dev/null 2>&1
+   if [ $? -ne 0 ]; then
+     echo aws user has no permission to eks
+     exit 1
+   fi
+
+  $AWS_CLI iam list-roles > /dev/null 2>&1
+   if [ $? -ne 0 ]; then
+     echo aws user has no permission to iam
+     exit 1
+   fi
+}
+
+function apply {
+  validate
+  mkdir -p $RUN_PATH
+  if [ $? -ne 0 ]; then
+     echo $RUN_PATH cannot be created
+     exit 1
+  fi
+  cp $CONFIG_PATH/*.tf $RUN_PATH
+  cd $RUN_PATH
+  $TERRAFORM init
+  if [ $? -ne 0 ]; then
+     exit 1
+  fi
+  echo "yes" | $TERRAFORM apply
+  echo eks cluster created.
+  $TERRAFORM output kubectl_config > $RUN_PATH/kubeconfig
+  echo run eks kubectl ... to acess it.
+}
+
+function destroy {
+  validate
+  if [ ! -d $RUN_PATH ]; then
+    exit 0
+  fi
+  cd $RUN_PATH
+  echo yes | TF_VAR_iam_instance_profile_name=$INST_PROFILE_NAME TF_VAR_cluster_iam_role_name=$ROLE_NAME $TERRAFORM destroy
+}
+
+function load {
+  image=$1
+  nodes=$(KUBECONFIG=$RUN_PATH/kubeconfig kubectl get nodes -o json | jq -r '.items[] | .status | .addresses[] | select(.type == "ExternalIP") |.address')
+  for node in $(echo $nodes); do
+    echo load $image to $node
+    docker save $image | bzip2 |pv| ssh  -oStrictHostKeyChecking=no ec2-user@$node 'docker load'
```

Comment:
add `pv` and `bzip2` to the required software in installation instructions

maybe add them to `CMD_ARRAY` so that the script checks that they are in the PATH

Reply:
added check for pv, jq and bzip2